src/PSRule.Rules.Azure/rules/Azure.Cognitive.Rule.yaml

```
@@ -22,9 +22,9 @@ spec:
   - Microsoft.CognitiveServices/accounts
   condition:
     anyOf:
-    - field: properties.publicNetworkAccess
+    - field: Properties.publicNetworkAccess
```

We shouldn't need to change this. Generally, I have been looking to updates these to start with a lower-case.

```suggestion
    - field: properties.publicNetworkAccess
      equals: Disabled
    - field: properties.networkAcls.defaultAction
```